src/Caching/Detector/ChangedFilesDetector.php

```diff
@@ -53,8 +51,7 @@ public function hasFileChanged(string $filePath): bool
         $cachedValue = $this->cache->load($fileInfoCacheKey, CacheKey::FILE_HASH_KEY);
 
         if ($cachedValue !== null) {
-            $currentFileHash = $this->hashFile($filePath);
-            return $currentFileHash !== $cachedValue;
+            return $fileInfoCacheKey !== $cachedValue;
```

Comment:
fileInfoCacheKey is cache key, while file hash is hashed of file, seems different

Reply:
it has the same content though

Reply:
- the `hash()` method using `hash()` function.

https://github.com/rectorphp/rector-src/blob/c59c83a99fdb883dbcf6fead62b7cd9961fae034/src/Util/FileHasher.php#L19

- the `hashes()` method using `hash_file()` funciton.

https://github.com/rectorphp/rector-src/blob/c59c83a99fdb883dbcf6fead62b7cd9961fae034/src/Util/FileHasher.php#L32

so the usage seems different ...

Reply:
I tested that `hash()` have different result vs `hash_file()`, see below:

```
➜  rector-src git:(main) php -a
Interactive shell

php > echo hash('xxh128', 'vendor/autoload.php');
ed4ee810129ade2015db3142c912be83

php > echo hash_file('xxh128', 'vendor/autoload.php');
d69f0ce7c725ec2c59342d08f047a9ad
```

so the change seems different.